bip122/caip2.md

```diff
@@ -60,11 +60,20 @@ This is a list of manually composed examples:
 # Bitcoin mainnet (see [chain ID section of BIP122](https://github.com/bitcoin/bips/blob/master/bip-0122.mediawiki#definition-of-chain-id))
 bip122:000000000019d6689c085ae165831e93
 
-# Litecoin
+# Litecoin mainnet
 bip122:12a765e31ffd4059bada1e25190f6e98
 
-# Feathercoin (Litecoin fork)
-bip122:fdbe99b90c90bae7505796461471d89a
```

Comment:
It would be nice to provide another example of a forked chain if Feathercoin is removed.

Perhaps Bitcoin Cash?

The split block is 478559. The first block after split is [478560](https://blockchair.com/bitcoin-cash/block/478560) with hash `000000000000000000b15ad892af8f6aca4462d46d0b6e5884cadc033c8f257b`.

That translates into chain ID `bip122:000000000000000000b15ad892af8f6a`

Reply:
great thinking, i'll add a commit for that